modules/ROOT/pages/publish-apis.adoc

```diff
@@ -4,114 +4,101 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-An ACM community administrator imports APIs, minor API updates, and major API updates from their Exchange organization to their community, reviews them, and publishes them.
+ACM community administrators import APIs, make minor and major API updates from the Exchange 
```

Comment:
This is incorrect. ACM comm admins import major and minor API updates that other people make in Exchange. ACM comm admins don't (usually) make major and minor API updates in Exchange.

Reply:
Fixed.